borg/platform_linux.pyx

```diff
@@ -5,6 +5,8 @@ import stat
 
 from .helpers import posix_acl_use_stored_uid_gid, user2uid, group2gid, safe_decode, safe_encode
 from .platform_base import SyncFile as BaseSyncFile
+from .platform_posix import swidth
+
```

Comment:
guess over time it'll become more. also, it avoids the temptation to just duplicate the code. :)